packages/zowe-explorer-ftp-extension/src/ZoweExplorerAbstractFtpApi.ts

```diff
@@ -109,11 +107,11 @@ export abstract class AbstractFtpApi implements ZoweExplorerApi.ICommon {
                         "Invalid Credentials. Please ensure the username and password for " +
                         validateProfile?.name +
                         " are valid or this may lead to a lock-out.";
-                    ZoweVsCodeExtension.showVsCodeMessage(errMsg, MessageSeverityEnum.ERROR, ZoweLogger);
+                    await Gui.errorMessage(errMsg, { logger: ZoweLogger });
                     throw new Error();
                 } else {
                     // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                    ZoweVsCodeExtension.showVsCodeMessage(e.message as string, MessageSeverityEnum.ERROR, ZoweLogger);
+                    await Gui.errorMessage(e.message as string, { logger: ZoweLogger });
```

Comment:
For consistency, should these two `errorMessage` calls be `void`? 😋

Reply:
`await` > `void` but sometimes `await` is needed for those `Thenable`s in synchronous functions, and would involve changing the function prototype to `async` and then `await`ing it everywhere it is used. 

Trying to a(void) `void` only because its not the ideal way of handling `Thenable`s/`async` calls, but is more of a convenience when a lot of synchronous functions reference the function that's using `void`, where usually that function needs to be changed to `async` for `await` to work.